src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.props

```diff
@@ -8,5 +8,13 @@
     <InformationalVersion Condition="'$(InformationalVersion)' == ''">$(AssemblyFileVersion)$(BuiltByString). $(SrcCodeRef)</InformationalVersion>
     <Product Condition="'$(Product)' == ''">Microsoft%AE .NET Framework</Product>
     <AssemblyTitle Condition="'$(AssemblyTitle)' == ''">$(AssemblyName)</AssemblyTitle>
+	<!-- File passed to C# compiler that defines the mapping from source to GIT URL.  Created in versioning.targets --> 
+    <SourceLinkFilePath>$(BaseIntermediateOutputPath)source_link.json</SourceLinkFilePath>
   </PropertyGroup>
+
+  <PropertyGroup Condition="Exists('$(SourceLinkFilePath)')">
```

Comment:
I think it makes sense to check `UseSourceLink` here too, just in case someone wants to turn off source linking but doesn't know to delete the file.